admin-ui/src/components/token-dialog.tsx

```
@@ -30,6 +35,19 @@ export function TokenDialog(): React.JSX.Element {
 
   const tokenStatus = token ? t("common.set") : t("common.unset")
 
+  useEffect(() => {
+    if (typeof window === "undefined") return
```

ESLint

[error] Prefer `globalThis.window` over `window`.

(unicorn/prefer-global-this)